composer.json

```diff
@@ -25,16 +25,15 @@
         "doctrine/doctrine-migrations-bundle": "^3.3.1",
         "doctrine/orm": "^2.19.6",
         "embed/embed": "^4.4.12",
-        "endroid/qr-code": "^4.8.5",
+        "endroid/qr-code": "^5.1.0",
         "friendsofsymfony/jsrouting-bundle": "^3.5.0",
         "furqansiddiqui/bip39-mnemonic-php": "^0.1.7",
         "gmostafa/php-graphql-client": "^1.13",
         "gumlet/php-image-resize": "^2.0.4",
         "knplabs/knp-time-bundle": "^2.4.0",
         "knpuniversity/oauth2-client-bundle": "^2.18.1",
         "kornrunner/blurhash": "^1.2.2",
-        "laminas/laminas-diactoros": "^2.26.0",
-        "landrok/activitypub": "^0.5.8",
+        "laminas/laminas-diactoros": "^3.3.1",
```

Comment:
Do we know what this is for?

Reply:
It CAN be used for: https://docs.laminas.dev/laminas-diactoros/v3/usage/#http-clients

But I don't we use it now?

Reply:
Maybe Symfony dropped support for it, years ago? https://github.com/laravel/framework/issues/31017

Reply:
Should I remove it as well ?

Reply:
Removed!